apps/kyb-app/src/common/components/layouts/Signup/Footer.tsx

```
@@ -0,0 +1,23 @@
+
+export const Footer: FunctionComponent<IFooterProps> = props => {
+  const { themeParams } = useSignupLayout();
+  const { rawHtml, styles } = themeParams?.footer || props || {};
+
+  if (!rawHtml) return null;
+
+  return (
+    <div
+      className="font-inter text-base text-[#94A3B8]"
+      style={styles}
+      dangerouslySetInnerHTML={{ __html: DOMPurify(window).sanitize(rawHtml) }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)